dash/frontend/src/app/modules/private/pages/external-auth-configuration/external-auth-configuration-create/external-auth-configuration-create.component.html

```diff
@@ -18,11 +18,11 @@ <h2 mat-dialog-title class="subnav-title">
       </mat-form-field>
     </p>
     <p *ngIf="oauthAuthActivated">
-      <mat-form-field (selectionchange)="onOAuthProviderTypeChange($event)">
+      <mat-form-field>
         <mat-label>Provider Type</mat-label>
-        <mat-select formControlName="providerType">
-          <mat-option value="GOOGLE">GOOGLE</mat-option>
-          <mat-option value="AZURE">Azure</mat-option>
+        <mat-select formControlName="providerType" (selectionChange)="onOAuthProviderTypeChange($event)">
+          <mat-option [value]="'GOOGLE'">Google</mat-option>
```

Comment:
Why was this change needed?

Reply:
The selection change was placed on the wrong tag previously, it needed to be moved

The capitalization on Google was changed for consistency with the Azure option